workspace/api/openfdd_bridge/plot_readings.py

```
@@ -163,16 +163,24 @@ def evaluate_fault_plots(
+        from open_fdd.arrow_runtime.rules import detect_rule_backend
+
+        table = pa.Table.from_pandas(df.reset_index(drop=True), preserve_index=False)
+        n = table.num_rows
+        if n == 0 or detect_rule_backend(code, rule) != "arrow":
+            continue
+        cfg = dict(rule.get("config") or {})
+        cfg.setdefault("site_id", site_id)
+        try:
+            arrow_result = run_arrow_rule(code, table, cfg, rule_id=rid)
+        except Exception:
             continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)